### PR TITLE
BAQE-1318 Implement Kie server scenarios with MySQL and PostgreSQL fo…

### PR DESCRIPTION
**Pull request by Sgitario**
This PR becomes a replacement for the old PR: https://github.com/kiegroup/kie-cloud-tests/pull/595

Jenkins job for MYSQL using operator: https://rhba-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/jcarvaja/job/jcarvaja-openshift-operator/36/

And for POSTGRESQL: https://rhba-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/jcarvaja/job/jcarvaja-openshift-operator/37

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerWithBuiltKjarIntegrationTest.java [hunk 1]
```diff
@@ -51,7 +52,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Category(Smoke.class)
+@Category({Smoke.class, OperatorNotSupported.class})
```
Comment (sutaakar) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerWithBuiltKjarIntegrationTest.java]: Why is operator not supported?

Comment (Sgitario) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerWithBuiltKjarIntegrationTest.java]: You asked me to unsupport the method withContainerDeployment in this comment: https://github.com/kiegroup/kie-cloud-tests/pull/595#discussion_r389621436

Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerWithBuiltKjarIntegrationTest.java]: Oh, right, I forgot on that comment.
It is correct.

File: framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/model/components/Server.java [hunk 2]
```diff
@@ -117,7 +117,7 @@ public Jms getJms() {
     public void setJms(Jms jms) {
         this.jms = jms;
     }
-
+
```
Comment (jakubschwan) [inline framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/model/components/Server.java]: remove white space
```suggestion

```

**Pull request merged**